spec/feature/authenticated_user_spec.rb

```diff
@@ -5,6 +5,12 @@
 
   let(:category1) { Category.create(name: "Breakfast") }
   let(:category2) { Category.create(name: "Lunch") }
+  let!(:valid_user) { User.create(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.